### PR TITLE
fixed shape data 404

### PR DESCRIPTION
**Pull request by lilyhahn**
fixed broken link by downloading data and adding to inst/extdata.

### CODE REVIEW

**Comment by lawinslow** [comment]
Could you bump the version number?

**Comment by lilyhahn** [comment]
fixed!

File: R/adk_data.R [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+#' Loads data from locally downloaded CSV files. Run check_dl_data before using this function.
+#' 
+#' @param data_name A string choosing the data to load. Can be "chem", "rotifer", "phyto", "crustacean", "tempdo", or "meta".
+#' 
+#' @export
+
+adk_data = function(data_name){
+  data_name = tolower(data_name)
+  path = local_path()
+  if(data_name == "chem"){
```
Comment (lawinslow) [inline R/adk_data.R]: Could you do this as a lookup table instead of a large else-if block?

**Comment by lawinslow** [comment]
Version bump too please!

**Review by lawinslow — APPROVED** [review]

**Pull request merged**